waspc/src/Wasp/EntityPSL.hs

```diff
@@ -0,0 +1,10 @@
+-- TODO(matija): remove PSL suffix, added it to avoid clashes with the existing Entity module.
+-- Once the old Entity module is removed, rename to "Entity".
+module Wasp.EntityPSL
+    ( EntityPSL (..)
+    ) where
+
+data EntityPSL = EntityPSL
+    { _name :: !String
+    , _pslModelSchema :: !String
```

Comment:
This should probably be Text? But ok no biggie.

Reply:
you are right! switched to Text